_posts/2020-06-14-dynamic-partition-pruning.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: post
+title:  "Dynamic partition pruning"
+author: Raunaq Morarka, Qubole and Karol Sobczak, Starburst Data
+excerpt_separator: <!--more-->
+---
+
```

Comment:
Maybe we need some kind of introduction here:
```
[Star-schema](https://en.wikipedia.org/wiki/Star_schema) is one of the most widely used data
mart pattern.
The star schema consist of fact tables (usually partitioned) and dimension tables,
which are used to filter rows from fact tables.
```

Reply:
Added